app/services/invoices/calculate_fees_service.rb

```diff
@@ -97,7 +97,11 @@ def create_charges_fees(subscription, boundaries)
     def should_create_subscription_fee?(subscription)
       # NOTE: When plan is pay in advance we generate an invoice upon subscription creation
       # We want to prevent creating subscription fee if subscription creation already happened on billing day
-      return false if subscription.plan.pay_in_advance? && subscription.fee_exists?(issuing_date)
+      fee_exists = subscription.fees.subscription_kind.where(created_at: issuing_date.beginning_of_day..issuing_date.end_of_day)
```

Comment:
Layout/LineLength: Line is too long. [128/120] (https://rubystyle.guide#max-line-length)